Makefile

```diff
@@ -15,3 +17,10 @@ extensions:
 
 clean:
 	$(MAKE) -C regression clean
+
+flake:
```

Comment:
My OCD cannot handle that `make flake` also executes black .. but using any other name is silly :P